oidc/client.go

```diff
@@ -656,12 +656,11 @@ func chooseAuthMethod(cfg ProviderConfig) (string, error) {
 }
 
 // SyncProviderConfig starts the provider config syncer
```

Comment:
It's not clear from the signature or the godoc what the new 2nd return value is for. Can you explain in the godoc?